### ISSUE TITLE
Request Zcash and BTG on Nvidia

### ISSUE THREAD

**Issue by fhorst1** [Optiminer/OptiminerEquihash#16]
I noticed that zcash and nvidia are not yet supported on optiminer.
MNX is and that is going great, but as the time passes, there will be a time that MNX is no longer profitable.

I have about 200 1060 cards.
Yes, I can go back to ethereum, but I like to continue to use optiminer.

If funds or a team for development is required, you can contact me.

Frank

**Comment by blisskidd** [comment]
If you want to mine equihash on nVidia, EWBF's miner is the thing to use.
Check out -> https://bitcointalk.org/index.php?topic=1707546.0

**Comment by fhorst1** [comment]
Nope, EWBF is NOT to use, NOT with the same performance like optiminer gives.
Have you tried EWBF on mnx? and on Optiminer?
Optiminer is by far more advanced and better (more) results.
